chainer/functions/util/forget.py

```diff
@@ -117,4 +117,8 @@ def forget(func, *xs):
         the method returns a tuple too.
 
     """
+    # Guarantee that backward is called by wrapping bare ndarrays as
+    # ``Variable`` s with ``requires_grad`` set to ``True``.
```

Comment:
How about adding this explanation to the docstring above? I think just adding a note to say it'll be wrapped by Variable if it's ndarray is fine.